SpatialGDK/Documentation/content/glossary.md

```diff
@@ -1,33 +1,325 @@
+### Interest
+There are two types of interest: entity interest and component interest.
+
+#### Entity interest
+A [worker](#workers) is interested in all [chunks](#chunk) that contain [entities](#spatialos-entity) it has
+[write access](#authority) to a [component](spatialos-componet) on. It's *also* interested in chunks within a configurable
+radius of those entities: this makes sure that workers are aware of entities nearby. You can set this radius
+in the [worker configuration file](#worker-configuration-file).
+If a worker is interested in a chunk, it will [check out](#check-out)all the entities in that chunk.
+
+#### Component interest
+Each [worker](#workers), in its [worker configuration file](#worker-configuration-file), specifies which [components](#components) it is
+interested in. SpatialOS only sends updates about components to a worker which is interested in that component.
+> Related:
+>
+> * [Entity interest settings]({{urlRoot}}/shared/worker-configuration/bridge-config#entity-interest)
+> * [Component delivery settings]({{urlRoot}}/shared/worker-configuration/bridge-config#component-delivery)
+
+
+### Launch
+>Not to be confused with [the launcher](#launcher).
+In SpatialOS, “launch” means start a game [deployment](#deployment). See also [launch configuration file](#launch-configuration-file).
+
+### Launch configuration
+The launch configuration is how you set up the start of your game’s [deployment](#deployment) (its launch). It is represented in the launch configuration file.
+See [workers](#workers) and [launch configuration file](#launch-configuration-file).
+
+### Launch configuration file
+The  [launch configuration file](#launch-configuration-file) is a `.json` file containing the information that the “launch a deployment” commands use to use to start a [deployment](#deployment).
+>Related:
+>
+>[Launch configuration file (SpatialOS documenation)](https://docs.improbable.io/reference/latest/shared/reference/file-formats/launch-config)
+
+### Launcher
+The Launcher is a tool that can download and start [game clients](#game-client) that connect to [cloud deployments](#deployment). It's available as an application for Windows and macOS. From the [Console](#console), you can use the Launcher to connect a game client to your own cloud deployment or generate a share link so anyone with the link can download a game client and join your game.
+The Launcher downloads the client executable from the [SpatialOS assembly](#assembly) you uploaded.
+> Related:
+>
+> * [The Launcher (SpatialOS documentation)](https://docs.improbable.io/reference/latest/shared/operate/launcher)
+### Load balancing
+One of the features of SpatialOS is load balancing: dynamically adjusting how many [components](#spatialos-components) on [entities](#spatialos-entities) in the [world](#spatialos-world) each [worker](#workers) has [write access](#authority) to, so that workers don’t get overloaded.
+Load balancing only applies to [server-workers](#workers).
```

Comment:
It looks like these were intended to be individual paragraphs, in which case they each need an extra line break.